File: backend/src/main/java/lan/dk/podcastserver/config/SchedulerConfig.java
package lan.dk.podcastserver.config;

import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.scheduling.annotation.EnableScheduling;

@Configuration
@EnableScheduling
@ComponentScan(basePackages = {"com.github.davinkevin.podcastserver.scheduled"})
public class SchedulerConfig {}

File: backend/src/main/kotlin/com/github/davinkevin/podcastserver/config/SchedulerConfig.kt
package com.github.davinkevin.podcastserver.config

import org.springframework.context.annotation.ComponentScan
import org.springframework.context.annotation.Configuration
import org.springframework.scheduling.annotation.EnableScheduling

@Configuration
@EnableScheduling
@ComponentScan(basePackages = arrayOf("com.github.davinkevin.podcastserver.scheduled"))
class SchedulerConfig
